### PR TITLE
Added multi-langauge support for bulk sensor upload

### PR DESCRIPTION
**Pull request by anant00729**
To test, 

Please check, https://lite-farm.atlassian.net/browse/LF-2659

### CODE REVIEW

**Review by dtrapplitefarm — APPROVED** [review]

File: packages/webapp/public/locales/fr/translation.json [hunk 1]
```diff
@@ -460,6 +460,21 @@
...
+        "READING_TYPES": "types_à_relevé",
+        "SENSOR_EXTERNAL_ID": "ID_externe",
+        "DEPTH": "Profondeur",
+        "BRAND": "Marque",
+        "MODEL": "Modèle"
+      },
+      "SENSOR_READING_TYPES": {
+        "SOIL_MOISTURE_CONTENT": "teneur_en_eau_du_sol",
+        "SOIL_WATER_POTENTIAL": "potential_hydrique_du_sol",
+        "TEMPERATURE": "temperatura"
```
Comment (Cali0707) [inline packages/webapp/public/locales/fr/translation.json:476]: Should be 'température'

**Review by Cali0707 — CHANGES_REQUESTED** [review]

File: packages/webapp/public/locales/fr/translation.json (hunk 1 above)
Comment (anant00729) [reply packages/webapp/public/locales/fr/translation.json]: Done

**Review by Cali0707 — APPROVED** [review]
Looks good, thanks for fixing that translation @anant00729 !

**Comment by Cali0707** [comment]
@anant00729 can you fix the merge conflict on this PR?

**Review by smattingly — APPROVED** [review]

**Pull request merged**